src/catalog/src/information_schema/memory_table/tables.rs

```diff
@@ -227,6 +227,187 @@ pub fn get_schema_columns(table_name: &str) -> (SchemaRef, Vec<VectorRef>) {
             vec![],
         ),
 
+        OPTIMIZER_TRACE => (
```

Comment:
Can we add a TODO here or create an issue for `OPTIMIZER_TRACE`? GreptimeDB already has query tracing, we can support it later cc @Taylor-lagrange